### ISSUE TITLE
Goodscents

### ISSUE THREAD

**Issue by rgerkin** [pyrfume/pyrfume-data#2]
http://www.thegoodscentscompany.com/allproc-1.html
http://www.thegoodscentscompany.com/categories.html
http://www.thegoodscentscompany.com/search2.html

**Comment by contrebande-labs** [comment]
Hi guys, so we are back on the data project. If we were to submit a PR for a csv dump of the "tgsc" data respecting the general structure and nomenclature of the csv files already included in this repository for other sources, would you accept it? After all this time we think here is the best place to host this data. And as far as "licensing issues" go, we think we should proceed with this being derivative work _original_ enough or at least _fair use_ enough for data that is already _publicly-accessible_. Either way, we think this course of action and point of view can be re-evaluated or even reversed should it ever be challenged by the putative authors of the prior art. What do you guys think? Please let us know and we will proceed with the tgsc dump PR. In  a similar fashion, we could also contribute a couple of other sources (some are on your wishlist, some aren't).

**Comment by rgerkin** [comment]
@contrebande-labs I think that is fair.  Thank you for the contribution.

**Comment by contrebande-labs** [comment]
Great! We have started dumping the [data in our fork](https://github.com/contrebande-labs/pyrfume-data/tree/master/tgsc). When all the data has been committed there in CSV format, we will issue a PR.

**Comment by contrebande-labs** [comment]
We have changed the data directory to ["goodscents"](https://github.com/contrebande-labs/pyrfume-data/tree/master/goodscents) to be consistent with how you have been naming this dataset in Pyrfume. Also, we have debated internally whether to include Goodscents' 3D conformer files (currently in the ["goodscents/cml"](https://github.com/contrebande-labs/pyrfume-data/tree/master/goodscents/cml) but we realized they were of very poor quality and it was better to get them from PubChem instead. We will be removing them soon.
